server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/EmailQuery.scala

```diff
@@ -35,6 +36,33 @@ object Limit {
 }
 case class QueryState(value: String) extends AnyVal
 
+sealed trait SortProperty {
+  def toSortClause: SortClause
+}
+case class ReceivedAtSortProperty() extends SortProperty {
+  override def toSortClause: SortClause = SortClause.Arrival
+}
+
+object IsAscending {
+  val default: IsAscending = IsAscending(false)
```

Comment:
Should not be name default, but descending. 
Descending order order is the default only if no comparator are provided, in this case we will use a comparator on Arrival with a descending sort order.
If a comparator is provided with no sort order specified, the default sort order is ascending